contents/blog/collaborating-on-posthog.md

```diff
@@ -0,0 +1,86 @@
+---
+date: 2022-03-24
+title: Collaborating on PostHog
```

Comment:
```suggestion
title: Introducing Collboration on PostHog
```

Up for debate whether it's 'Collaboration' or 'collaboration' in this context, but I'd like most of these posts to follow headline convention.